spec/features/user_edits_voter_spec.rb

```diff
@@ -24,7 +24,7 @@
     fill_in "State", with: "MA"
     fill_in "Zip", with: "01234"
 
-    click_button 'Submit'
+    click_button 'Update Voter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.